packages/block-library/src/navigation/index.php

```diff
@@ -254,37 +219,9 @@ function block_core_navigation_render_submenu_icon() {
  * @return object WP_Term The classic navigation.
  */
 function block_core_navigation_get_classic_menu_fallback() {
-	$classic_nav_menus = wp_get_nav_menus();
-
-	// If menus exist.
-	if ( $classic_nav_menus && ! is_wp_error( $classic_nav_menus ) ) {
-		// Handles simple use case where user has a classic menu and switches to a block theme.
+	_deprecated_function( __FUNCTION__, '5.8.0', 'WP_Navigation_Gutenberg::get_classic_menu_fallback' );
```

Comment:
What version am I supposed to put in here?

Reply:
6.3